src/Slices/ServiceDetails/UI/Tabs/Callbacks/Row.tsx

```diff
@@ -29,25 +29,35 @@ export const Row: React.FC<Props> = ({
   return (
     <Tbody isExpanded={false}>
       <Tr aria-label={`CallbackRow-${shortUuid}`}>
-        <Td>{callback.url}</Td>
-        <Td>
-          <TextWithCopy
-            value={callback.callback_id}
-            tooltipContent={words("catalog.callbacks.uuid.copy")}
-          >
-            {shortUuid}
-          </TextWithCopy>
-        </Td>
-        <Td>{callback.minimal_log_level}</Td>
         <Td>
-          {callback.event_types && (
-            <ListWithTeaser
-              items={callback.event_types}
+          {callback.event_types ? (
+            <Toggle
+              text={callback.url}
               onClick={onToggle}
               isExpanded={isExpanded}
             />
+          ) : (
+            <StyledText>{callback.url}</StyledText>
           )}
         </Td>
+        <Td>
+          <Tooltip content={callback.callback_id}>
+            <TextWithCopy
+              value={callback.callback_id}
+              tooltipContent={words("catalog.callbacks.uuid.copy")}
+            >
+              {shortUuid}
+            </TextWithCopy>
+          </Tooltip>
+        </Td>
+        <Td>{callback.minimal_log_level_text}</Td>
+        <Td>
+          {callback.event_types &&
```

Comment:
Are you ellypsing ? Or what's the idea of this?

Reply:
That was previous approach to truncate chosen event_type, it was in ListWithTeaser, but that component was redundant after changing position of the toggle

Reply:
I don't know for you, but I don't like the way it's displayed. Since they are shown in the expansion, what about displaying in the table : "X Event Types" and the tooltip/expansion will show the full selection? 
@bartv what is your thought on this? The ellypsing of the list means bitterly nothing to the user, the first name isn't even fully readable and the user is forced anyway to click open the row.